### ISSUE TITLE
"Abandon" grunt

### ISSUE THREAD

**Issue by samreid** [phetsims/chipper#1436]
Grunt is an older build system that puts a lot of constraints on our system, and that we aren't leveraging too much. While discussing https://github.com/phetsims/chipper/issues/1272, @zepumph and I brainstormed a way we could abandon grunt without breaking backward compatibility:

* Keep grunt as an entry point
* Forward grunt commands to a new script via spawn or child process.

There is a sketch about how this could work in https://github.com/phetsims/chipper/commit/1c7d111d9919c6130f41b1d595c40b518bdb0d76

We do not want this to be in-scope for https://github.com/phetsims/chipper/issues/1272, but wanted to write it down for later.

**Comment by zepumph** [comment]
Yes thanks so much! It will be excellent to schedule this work once we have a better idea of how we are going to support typescript in our build tools.

**Comment by zepumph** [comment]
Perhaps a separate issue to work on that is best.

**Comment by marlitas** [comment]
Our hypothesis is that abandoning grunt will solve a lot of the dependabot alerts: https://github.com/phetsims/special-ops/issues/341

**Comment by jonathanolson** [comment]
Lots of my changes in https://github.com/phetsims/totality/issues/140 are reducing or removing usage of grunt. Won't go for the older release branches, but progressing toward not using it at all.
